### DISCUSSION TITLE
이거 풀게요 12월 마지막 주

### DISCUSSION THREAD

**Discussion by csj4032** [brewagebear/ISOLVEDTHEPROBLEM#28] [금주의 문제]
[올바른 괄호](https://programmers.co.kr/learn/courses/30/lessons/12909)
[타겟 넘버](https://programmers.co.kr/learn/courses/30/lessons/43165)

[N개의 최소공배수](https://programmers.co.kr/learn/courses/30/lessons/12953)

**Comment by brewagebear** [comment]
재귀를 연습하는데 도움이 될 수 있는 추가적인 문제들 등록해봅니다.

+ 릿코드 
[binary-tree-inorder-traversal](https://leetcode.com/problems/binary-tree-inorder-traversal/)
[path-sum](https://leetcode.com/problems/path-sum/)
[binary-tree-level-order-traversal](https://leetcode.com/problems/binary-tree-level-order-traversal/)
[flood-fill](https://leetcode.com/problems/flood-fill/)

+ 백준
[N과M 시리즈](https://www.acmicpc.net/workbook/view/2052)

**Comment by Tianea2160** [comment]
https://programmers.co.kr/learn/courses/30/lessons/43165
https://programmers.co.kr/learn/courses/30/lessons/60058

**Comment by abcdsds** [comment]
저는 이번주 스킵하고 재귀 기본문제 풀겠습니다.
